docs/core/tools/dotnet-restore.md

```diff
@@ -43,7 +43,22 @@ For dependencies, you specify where the restored packages are placed during the
 
 For project-specific tooling, `dotnet restore` first restores the package in which the tool is packed, and then proceeds to restore the tool's dependencies as specified in its project file.
 
-The behavior of the `dotnet restore` command is affected by some of the settings in the *Nuget.Config* file, if present. For example, setting the `globalPackagesFolder` in *NuGet.Config* places the restored NuGet packages in the specified folder. This is an alternative to specifying the `--packages` option on the `dotnet restore` command. For more information, see the [NuGet.Config reference](/nuget/schema/nuget-config-file).
+### Nuget.Config differences
+The behavior of the `dotnet restore` command is affected by the settings in the *Nuget.Config* file, if present. For example, setting the `globalPackagesFolder` in *NuGet.Config* places the restored NuGet packages in the specified folder. This is an alternative to specifying the `--packages` option on the `dotnet restore` command. For more information, see the [NuGet.Config reference](/nuget/schema/nuget-config-file).
+
+There are three specific settings that `dotnet restore` ignores.
+
+* [bindingRedirects](/nuget/schema/nuget-config-file#bindingredirects-section)
+
+  Binding redirects does not work with `<PackageReference>` elements and .NET Core only supports `<PackageReference>` elements for NuGet packages.
+
+* [solution](/nuget/schema/nuget-config-file#solution-section)
+
+  This setting is only Visual Studio specific and no longer applies to the latest version of Visual Studio.
```

Comment:
Is it worth/possible to mention the concrete version of Visual Studio, starting with which the setting doesn't apply anymore?

Reply:
I'll see if I can find that info. I was just told "it no longer applies"

Reply:
I clarified the real issue. "Legacy" was referring to the old project style that used packages.config. I just removed that wording and clarified what the difference really is.